Prepare for Quarkus 3.0.0

In the advent of the Quarkus 3.0.0 release, we should prepare this extension.

We depend on [`quarkus-artemis-jms`](https://github.com/quarkiverse/quarkus-artemis), therefore we need builds of this extension in order to test compatibility.

Current release of `quarkus-artemis-jms`:[![Version](https://img.shields.io/maven-central/v/io.quarkiverse.artemis/quarkus-artemis-jms)](https://search.maven.org/artifact/io.quarkiverse.artemis/quarkus-artemis-jms)

It seems that narayana has not yet been migrated to jakartaEE:

```
[ERROR] /eXchange/git/quarkiverse/quarkus-pooled-jms/runtime/src/main/java/io/quarkiverse/messaginghub/pooled/jms/PooledJmsWrapper.java:[48,28] cannot access javax.transaction.TransactionManager
  class file for javax.transaction.TransactionManager not found
[ERROR] /eXchange/git/quarkiverse/quarkus-pooled-jms/runtime/src/main/java/io/quarkiverse/messaginghub/pooled/jms/PooledJmsWrapper.java:[54,58] cannot access jakarta.jms.XAConnectionFactory
  class file for jakarta.jms.XAConnectionFactory not found
```

--- a/integration-tests/src/main/java/io/quarkiverse/messaginghub/pooled/jms/it/PooledJmsResource.java
+++ b/integration-tests/src/main/java/io/quarkiverse/messaginghub/pooled/jms/it/PooledJmsResource.java
@@ -16,19 +16,19 @@
  */
 package io.quarkiverse.messaginghub.pooled.jms.it;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
-import javax.jms.ConnectionFactory;
-import javax.jms.JMSConsumer;
-import javax.jms.JMSContext;
-import javax.jms.JMSException;
-import javax.jms.JMSProducer;
-import javax.jms.Message;
-import javax.transaction.TransactionManager;
-import javax.transaction.Transactional;
-import javax.ws.rs.GET;
-import javax.ws.rs.POST;
-import javax.ws.rs.Path;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.jms.ConnectionFactory;
+import jakarta.jms.JMSConsumer;
+import jakarta.jms.JMSContext;
+import jakarta.jms.JMSException;
+import jakarta.jms.JMSProducer;
+import jakarta.jms.Message;
+import jakarta.transaction.TransactionManager;
+import jakarta.transaction.Transactional;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
 
 @Path("/pooled-jms")
 @ApplicationScoped

--- a/pom.xml
+++ b/pom.xml
@@ -20,9 +20,9 @@
     <maven.compiler.release>11</maven.compiler.release>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <quarkus.version>2.16.1.Final</quarkus.version>
-    <quarkus-artemis.version>2.0.4</quarkus-artemis.version>
-    <pooled-jms.version>2.0.5</pooled-jms.version>
+    <quarkus.version>3.0.0.Alpha3</quarkus.version>
+    <quarkus-artemis.version>3.0.0.Alpha3</quarkus-artemis.version>
+    <pooled-jms.version>3.1.0</pooled-jms.version>
   </properties>
   <dependencyManagement>
     <dependencies>

--- a/runtime/src/main/java/io/quarkiverse/messaginghub/pooled/jms/PooledJmsWrapper.java
+++ b/runtime/src/main/java/io/quarkiverse/messaginghub/pooled/jms/PooledJmsWrapper.java
@@ -1,7 +1,7 @@
 package io.quarkiverse.messaginghub.pooled.jms;
 
-import javax.jms.ConnectionFactory;
-import javax.transaction.TransactionManager;
+import jakarta.jms.ConnectionFactory;
+import jakarta.transaction.TransactionManager;
 
 import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory;
 import org.eclipse.microprofile.config.ConfigProvider;

